tests/system/general/test_08_Primitives3D.py

```diff
@@ -2110,3 +2110,15 @@ def test_96_sweep_around_axis(self):
         assert circle1.name in self.aedtapp.modeler.solid_names
         assert circle2.name in self.aedtapp.modeler.solid_names
         assert circle3.name in self.aedtapp.modeler.solid_names
+
+    def test_97_uncover_faces(self):
+        o1 = self.aedtapp.modeler.create_circle(cs_plane=0, position=[0, 0, 0], radius=10)
+        assert self.aedtapp.modeler.uncover_faces([o1.faces[0]])
+        c1 = self.aedtapp.modeler.create_circle(
+            orientation=ansys.aedt.core.constants.AXIS.X,
+            origin=[0, 10, 20],
+            radius="3",
+            name="Circle1",
+        )
+        b1 = self.aedtapp.modeler.create_box(origin=[-13.9, 0, 0], sizes=[27.8, -40, 25.4], name="Box1")
+        assert self.aedtapp.modeler.uncover_faces([c1.faces[0], b1.faces[0], b1.faces[2]])
```

Comment:
What I meant here, is that after this operation, you could check that the number of faces of c1,  is different than at the beginning...